### ISSUE TITLE
Cannot navigate to commands as buffer dynamically updates

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#177923]
From @Tyriar:

- Fill the scrollback
- Run some command that outputs over time
- Focus accessible buffer
- Navigate commands via the quick pick, _bufferToEditorIndex will point at the original stale values

**Comment by eleanorjboyd** [comment]
How is this command done? "Navigate commands via the quick pick, _bufferToEditorIndex will point at the original stale values". Not sure what `_bufferToEditorIndex` is, thanks!

**Comment by meganrogge** [comment]
It's expected to navigate to the command you currently have active in the quickpick like what is shown here
https://user-images.githubusercontent.com/29464607/234960843-08bc20b1-e4ad-4131-afbf-a6803e0ee617.mov
